src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -75,7 +75,7 @@ public enum ErrorCode {
   SHRED_DOC_LIMIT_VIOLATION("Document size limitation violated"),
 
   EXISTING_COLLECTION_DIFFERENT_SETTINGS("Collection already exists"),
-
+  INVALID_DATA_API_COLLECTION("Invalid Data API collection"),
```

Comment:
Should this be named similar to above one `EXISTING_COLLECTION_...` as maybe `EXISTING_TABLE`: it is not really a Collection at all but a Table; and we cannot try to re-create it.